runners/jstorm/pom.xml

```diff
@@ -53,18 +53,29 @@
                   <goal>test</goal>
                 </goals>
                 <configuration>
+                  <!--<includes>
+                    <include>org.apache.beam.sdk.transforms.CombineTest.java</include>
```

Comment:
<!--new_thread; commit:d8cb47c12a014273ba68adce0042f23f9e3e529e; resolved:0-->
Delete this?